packages/camera/tizen/src/log.h

```diff
@@ -8,9 +8,14 @@
 #endif
 #define LOG_TAG "CameraPlugin"
 
-#define LOG(prio, fmt, arg...)                                                 \
-  dlog_print(prio, LOG_TAG, "%s: %s(%d) > " fmt, __FILE__, __func__, __LINE__, \
-             ##arg)
+#ifndef __MODULE__
```

Comment:
@swift-kim if you still have a plan what keep a log template for plugin, I think it would be nice to add ```__MODULE__``` macro

Reply:
Ah, yes, I'm planning not to remove the log.h file from the plugin template and instead just wrap the plugin class with an unnmaed namespace:

https://github.com/swift-kim/flutter-tizen/commit/ac78aacac77d53f6f821daa5fb7fe2ff64ce3d55

Isn't `__MODULE__` defined by default?

Reply:
Well, I thought there was, but there was no.